### ISSUE TITLE
Could you share the dataset?

### ISSUE THREAD

**Issue by SailorKevin** [luoda888/CCF2018-Top2-Demand-Forecast#3]
We are intereted in your work. Could you share the original dataset for me, including the training set and test set.

Thank you so much.

**Comment by SailorKevin** [comment]
And my email: ydlkevin@gmail.com

Thank you for your contribution.
